client/mm/libxc/binance.go

```diff
@@ -1604,6 +1754,10 @@ func (bnc *binance) connectToMarketDataStream(ctx context.Context, baseID, quote
 			PingWait: time.Minute * 4,
 			ReconnectSync: func() {
 				bnc.log.Debugf("Binance reconnected")
+				select {
+				case reconnectC <- struct{}{}:
+				default:
+				}
 			},
 			ConnectEventFunc: func(cs comms.ConnectionStatus) {},
```

Comment:
Whenever this function is called with `Disconnected`, we should set all the orderbooks to be unsynced, so that the bots don't keep placing orders.

Reply:
desyncing books on disconnect now